### ISSUE TITLE
[BUG] Unable to set 'access_token' on api Configuration

### ISSUE THREAD

**Issue by fbl100** [OpenAPITools/openapi-generator#13315]
#### Bug Report Checklist

- [x ] Have you provided a full/minimal spec to reproduce the issue?
- [ x] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [ x] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [ x] Have you searched for related issues/PRs?
- [ x] What's the actual output vs expected output?
- [ ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

##### Description
This is for the `python-experimental` client generator.

When working with a 'bearer' access token, I am unable to set the 'access_token' property in the Configuration() initializer.  Instead, I have to do this:
```python
configuration = client_api.Configuration(
    host=URL,
)
configuration.access_token = getToken()
```

ideally I would be able to do this:
```python
configuration = client_api.Configuration(
    host=URL,
    access_token = getToken()
)
```
##### openapi-generator version
latest master (8/30/2022)

##### OpenAPI declaration file content or url
N/A

**Comment by wtrocki** [comment]
CC @dimakis

**Comment by fbl100** [comment]
I tend to agree, but won't have the bandwidth for a fix until mid Sept.  I wrote the issue so that it doesn't get forgotten about and would be happy to fix when I have time.  The workaround is easy enough for now.

**Comment by labrown** [comment]
I just ran into this bug today.  I see this [bit of code ](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/python/configuration.mustache#L186-L200
) and it is clear that setting access_token during the init of a Configuration object will be overwritten to 'None'.  I am not familiar enough with the codebase to know the correct fix for this, but it should be simple.

```
        self.access_token = access_token
        """Access token
        """
{{#hasHttpSignatureMethods}}
        if signing_info is not None:
            signing_info.host = host
        self.signing_info = signing_info
        """The HTTP signing configuration
        """
{{/hasHttpSignatureMethods}}
{{#hasOAuthMethods}}
        self.access_token = None
        """access token for OAuth/Bearer
        """
{{/hasOAuthMethods}}
```

**Comment by Neudrino** [comment]
Our current workaround is to explicitly set the token in the override headers of each request.

Setting the token statically in the configuration would not be sufficient for us, as the token expire frequently.
I.e. for the token to be set in the API client configuration, we would need a lazy evaluation, i.e. some sort of Promise to be set, such that the actual token value is only obtained once the request is made, such that in case a token refresh can be triggered before using the token.

**Comment by Neudrino** [comment]
This ticket might be related to:
- https://github.com/OpenAPITools/openapi-generator/issues/10688
- https://github.com/OpenAPITools/openapi-generator/issues/3143
